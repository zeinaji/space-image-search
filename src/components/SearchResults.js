import React from "react";

const SearchResults = (props) => {
  return (
    <>
      <img className="search-image" src={props.image} alt="search-result" />
    </>
  );
};

export default SearchResults;
