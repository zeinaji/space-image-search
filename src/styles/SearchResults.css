.search-image {
  display: flex;
  flex-wrap: wrap;
  margin: 20px 40px 40px 40px;
  justify-content: center;
  width: 100%;
  padding: 0;
}

.image-container {
  height: 200px;
  width: 200px;
  margin: 10px 30px 10px 30px;
}

.image-result {
  width: 100%;
  height: 100%;
  border-radius: 15px;
}
