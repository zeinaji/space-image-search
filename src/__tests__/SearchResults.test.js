import React from "react";
import { render, screen } from "@testing-library/react";
import SearchResults from "../components/SearchResults";

describe("SearchResults", () => {
  render(<SearchResults />);
  xit("it renders correctly", () => {
    const { asFragment } = render(<SearchResults image="mockImage" />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("it renders an image", () => {
    expect(screen.getByAltText("search-result")).toHaveClass("search-image");
  });
});
