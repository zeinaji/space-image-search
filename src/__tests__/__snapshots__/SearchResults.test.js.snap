// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults it renders correctly 1`] = `
<DocumentFragment>
  <img
    alt="search-result"
    src="mockImage"
  />
</DocumentFragment>
`;
